src/test/pythonnode_test.cpp

```diff
@@ -479,3 +442,404 @@ TEST_F(MediapipePythonNodeTest, PythonNodePassArgumentsToConstructor) {
+        tensor.numElements = numElements;
+        tensor.size = numElements * sizeof(T);
+        tensor.shape = std::vector<py::ssize_t>{1, numElements};
+        getPythonBackend()->createOvmsPyTensor(tensor.name, (void*)tensor.data, tensor.shape, tensor.datatype, tensor.size, tensor.pyTensor);
+        return tensor;
+
+    }
+
+    static std::vector<T> readVectorFromOutput(const std::string& outputName, int numElements, const mediapipe::CalculatorRunner* runner) {
+        const PyObjectWrapper<py::object>& pyOutput = runner->Outputs().Tag(outputName).packets[0].Get<PyObjectWrapper<py::object>>();
+        T * outputData = (T*)pyOutput.getProperty<void*>("ptr");
+        std::vector<T> output;
+        output.assign(outputData, outputData + numElements);
+        return output;
+    }
+
+    std::vector<T> getIncrementedVector() {
+        std::vector<T> output;
+        T * fpData = (T*)data;
+        for (int i = 0; i < shape[1]; i++) {
+            output.push_back(fpData[i] + 1);
+        }
+        return output;
+    }
+};
+
+// ---------------------------------- OVMS deserialize and serialize tests
+
+// This part duplicates some parts of tests in mediapipeflow_test.cpp file. 
+// We could think about moving them there in the future, but for now we need to keep 
+// all tests involving Python interpreter in a single test suite. 
+
+class MockedMediapipeGraphExecutorPy : public ovms::MediapipeGraphExecutor {
+public:
+    Status serializePacket(const std::string& name, ::inference::ModelInferResponse& response, const ::mediapipe::Packet& packet) const {
+        return ovms::MediapipeGraphExecutor::serializePacket(name, response, packet);
+    }
+
+    MockedMediapipeGraphExecutorPy(const std::string& name, const std::string& version, const ::mediapipe::CalculatorGraphConfig& config,
+        stream_types_mapping_t inputTypes,
+        stream_types_mapping_t outputTypes,
+        std::vector<std::string> inputNames, std::vector<std::string> outputNames,
+        const std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>>& pythonNodeResources,
+        PythonBackend* pythonBackend) :
+        MediapipeGraphExecutor(name, version, config, inputTypes, outputTypes, inputNames, outputNames, pythonNodeResources, pythonBackend) {}
+};
+
+
+TEST_F(PythonFlowTest, SerializePyObjectWrapperToKServeResponse) {
+    ovms::stream_types_mapping_t mapping;
+    mapping["python_result"] = mediapipe_packet_type_enum::OVMS_PY_TENSOR;
+    const std::vector<std::string> inputNames;
+    const std::vector<std::string> outputNames;
+    const ::mediapipe::CalculatorGraphConfig config;
+    std::unordered_map<std::string, std::shared_ptr<PythonNodeResource>> pythonNodeResources;
+    auto executor = MockedMediapipeGraphExecutorPy("", "", config, mapping, mapping, inputNames, outputNames, pythonNodeResources, getPythonBackend());
+
+    std::string datatype = "FP32";
+    std::string name = "python_result";
+    int numElements = 3;
```

Comment:
```
const int numElements = 3;
float input[numElements] = {...
```
This will create compiler error when number of elements do not match

Reply:
Is it worth changing in such simple scenario? Any suggestions?